### DISCUSSION TITLE
questions about a few magic numbers

### DISCUSSION THREAD

**Discussion by CSEEduanyu** [sgl-project/sglang#2296] [General]
(https://github.com/sgl-project/sglang/blob/5f12f0e7af585973b366c262a5c3faea5ce5bf0a/python/sglang/srt/model_executor/model_runner.py#L502)

**Comment by CSEEduanyu** [comment]
512 2048 4096
What I want to know is why these three numbers are different?

**Comment by zhaochenyang20** [comment]
Could you describe the problem more formally.
